doc/Language/syntax.pod6

```diff
@@ -327,52 +329,64 @@ so C<infix:<+>> and C<infix:<->> are two different operators. The bracketing
 characters used, however, do not count as part of the name; only the quoted data
 matters. So these are all the same name:
 
+=begin code :skip-test
     infix:<+>
     infix:<<+>>
     infix:«+»
     infix:['+']
     infix:('+')
+=end code
 
 Similarly, all of this works:
 
+=begin code :skip-test
```

Comment:
Why do you skip text here? You should do that only if it's not syntactically correct. Also, whenever you add that attribute, you should say why you are skipping the test

Reply:
I added the :skip-tests because one of the GitHub-checks failed, and it wasn't clear to me what caused the issue. Adding the :skip-tests solved the issue though. - I have now removed them all again. So next time one of the checks fails, maybe you can help me identify the issue.